numba/tests/test_parfors.py

```diff
@@ -24,7 +24,7 @@
 import subprocess as subp
 
 import numba.parfors.parfor
-from numba import njit, prange, set_num_threads, get_num_threads, typeof
+from numba import njit, prange, set_num_threads, get_num_threads, typeof, literally
```

Comment:
```suggestion
from numba import njit, prange, set_num_threads, get_num_threads, typeof
```
think `literally` is now unused?